### ISSUE TITLE
Document cache in the UI

### ISSUE THREAD

**Issue by danrabbit** [elementary/houston#410]
We should document in the UI somewhere that it may take up to an hour for new releases or new repos to be discovered in the dashboard

**Comment by btkostner** [comment]
I agree this should be noted, but this agressive cache should hopefully be a temporary fix until the GitHub process is streamlined.

**Comment by btkostner** [comment]
Is this going to be after we move the markdown docs over to houston?
